vllm/model_executor/models/internvl.py

```diff
@@ -473,6 +473,12 @@ def dummy_data(
 @INPUT_REGISTRY.register_input_processor(input_pipeline.input_processor)
 class InternVLChatModel(nn.Module, SupportsMultiModal, SupportsPP):
 
+    bitsandbytes_stacked_params_mapping = {
+        # shard_name, weight_name, index
+        "w1": ("gate_up_proj", 0),
+        "w3": ("gate_up_proj", 1),
+    }
```

Comment:
This is only compatible with InternLM2 backbone. However, InternVL2.5 and InternVL2-76B are using `Qwen2.5` and `Llama-3.1` as LLM backbone respectively.

Reply:
Great point -- I was actually already able to use BNB on the 4B  / Qwen2-based InternVL 2.5 without issue. I'll see if I can finesse this into something that better handles the dual backbones over the next few days. Thanks.